peer/main.go

```diff
@@ -244,15 +244,22 @@ func main() {
 	flags := nodeStartCmd.Flags()
 	flags.BoolVarP(&chaincodeDevMode, "peer-chaincodedev", "", false, "Whether peer in chaincode development mode")
 
+	var alternativeCfgPath = os.Getenv("PEER_CFG_PATH")
+	if alternativeCfgPath != "" {
```

Comment:
@gabre I think the whitelist function is ok for now.  Please do readd the noop bdd test and mark the issue resolved.

I think the other option for the whitelist is to keep a separate file in the system chaincode folder just for the whitelist. Will add a comment and an issue for that. That can be treated separate from this PR.